src/connectors/userService.ts

```diff
@@ -600,7 +600,49 @@ export class UserService extends BaseService {
     viewerId?: string | null
     exclude?: GQLSearchExclude
   }) => {
-    return { nodes: [], totalCount: 0 }
+    const displayName = key
+    const userName =
+      key.startsWith('@') || key.startsWith('＠') ? key.slice(1) : key
+
+    if (!userName) {
+      return { nodes: [], totalCount: 0 }
+    }
+
+    const baseQuery = this.knex('search_index.user')
+      .select('id')
+      .whereLike('display_name', `%${displayName}%`)
+      .orWhereLike('user_name', `%${userName}%`)
+      .orderByRaw('display_name = ? DESC', [displayName])
+      .orderByRaw('user_name = ? DESC', [userName])
+      .orderByRaw('display_name ~ ? DESC', [displayName])
+      .orderByRaw('num_followers DESC NULLS LAST')
```

Comment:
Same question as above, shall we `.limit` the query results?

Reply:
limit is set in L639